README.md

```diff
@@ -38,13 +38,13 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1** Tanzila
 
-- GitHub: [@tanzila-ebedines](https://github.com/githubhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/linkedinhandle)
+- GitHub: [@tanzila-abedin](https://github.com/tanzila-abedin)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tanzila-abedin-331440b2/)
 
 👤 **Author2** Shoira
 
-- GitHub: [@shoirata](https://github.com/githubhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/linkedinhandle)
+- GitHub: [@shoirata](https://github.com/ShoiraTa)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/shoira-tashpulatova-bab4a7122)
```

Comment:
## \[Optional\]: Improve the LinkedIn links 
- [ ]  Great job fixing the social links. I would advise if possible, we remove the Author tags and just have only our names in these sections since the title is already there in the heading of this section in this `README. and it would be much neater as well.  What are your thoughts on this? Thank you!

```markdown
👤 **Tanzila**

 GitHub: [@tanzila-abedin](https://github.com/tanzila-abedin)
- LinkedIn: [Tanzila Abedin](https://www.linkedin.com/in/tanzila-abedin-331440b2/)

👤 **Shoira**

- GitHub: [@shoirata](https://github.com/ShoiraTa)
- LinkedIn: [Shoira Tashpulatova](https://www.linkedin.com/in/shoira-tashpulatova-bab4a7122)

```

Reply:
Thank you  @christianotieno .  👍🏽 
I have removed those author tags as you suggested.